mcp_server/server.py

```
@@ -0,0 +1,966 @@
+# METRICS TOOLS (4)
+# =============================================================================
+
+
+@mcp.tool()
+@handle_mcp_error
+async def record_metrics(
+    task_id: str,
+    response_time: float,
+    success: bool,
+    token_usage: int = 0,
+    error_type: Optional[str] = None,
```

Ruff (0.14.10)

Unused function argument: `error_type`

(ARG001)